### PR TITLE
Add new volunteer ibukunoluwanap

### PR DESCRIPTION
**Pull request by ibukunoluwanap**

### CODE REVIEW

File: data/users/ibukunoluwanap.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Ibukunoluwa Naphtali",
+  "bio": "💻 • any software development | 🥗 • gets better with food | 🎧 • while listening to beats | 🕹️ • after the game night | creator of flutterme.org",
```
Comment (RakeshSangem) [inline data/users/ibukunoluwanap.json:3]: Hey @ibukunoluwanap , please use short description for example `Frontend developer` `Java developer` `Data Analyst`.

File: data/users/ibukunoluwanap.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  "image": "https://avatars.githubusercontent.com/u/61067520",
+  "categories": [
+    {
+      "slug": "programming | software-development | web-development | mobile-development | data-science",
```
Comment (RakeshSangem) [inline data/users/ibukunoluwanap.json:7]: You need to use only one slug from `public/categories.json`

**Review by RakeshSangem — CHANGES_REQUESTED** [review]

File: data/users/ibukunoluwanap.json (hunk 2 above)
Comment (ibukunoluwanap) [reply data/users/ibukunoluwanap.json:7]: Okay on it

**Comment by ibukunoluwanap** [comment]
Updated as requested

File: data/users/ibukunoluwanap.json [hunk 3]
```diff
@@ -1,10 +1,10 @@
 {
   "name": "Ibukunoluwa Naphtali",
-  "bio": "💻 • any software development | 🥗 • gets better with food | 🎧 • while listening to beats | 🕹️ • after the game night | creator of flutterme.org",
+  "bio": "A software developer working with Flutter, ReactJS and NodeJS. Creator of flutterme.org",
```
Comment (RakeshSangem) [inline data/users/ibukunoluwanap.json:3]: Please choose short description like `Web Developer` , `Data Analyst` , `Frontend dev`

**Review by ibukunoluwanap — COMMENTED** [review]
updated bio

File: data/users/ibukunoluwanap.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Ibukunoluwa Naphtali",
+  "bio": "Software developer, Frontend developer, Designer",
```
Comment (codewithvoid) [inline data/users/ibukunoluwanap.json:3]: Prefer having one title here

File: data/users/ibukunoluwanap.json [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+  "image": "https://avatars.githubusercontent.com/u/61067520",
+  "categories": [
+    {
+      "slug": "software-development",
+      "topics": ["Frontend","Flutter", "ReactJS", "NodeJS"]
```
Comment (codewithvoid) [inline data/users/ibukunoluwanap.json:8]: Frontend is in your title.

Flutter should go in `mobile-development` category

ReactJS and NodeJS in `web-development` category

Final file should be like this

```json
{
  "categories": [
    {
      "slug": "web-development",
      "topics": [
        "ReactJS",
        "NodeJS"
      ]
    },
    {
      "slug": "mobile-development",
      "topics": [
        "Flutter"
      ]
    }
  ]
}
```

**Comment by codewithvoid** [comment]
Thanks for volunteering @ibukunoluwanap :)

Added some comments

**Review by ibukunoluwanap — COMMENTED** [review]
updated categories

**Review by codewithvoid — APPROVED** [review]

**Comment by ibukunoluwanap** [comment]
thanks. check https://github.com/codewithvoid/AskDevs/pull/57/

**Pull request merged**